lib/charms/mongodb/v0/mongodb_backups.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""In this class, we manage client database relations.
+
+This class creates a user and database for each application relation
+and expose needed information for client connection via fields in
+external relation.
+"""
```

Comment:
It seems that this description is the same as the one from https://github.com/canonical/mongodb-operator/blob/main/lib/charms/mongodb/v0/mongodb_provider.py#L4, and not related to backups.

Reply:
I am embarrassed about this 🤦 I will correct this asap